Source/Particles/LaserParticleContainer.cpp

```diff
@@ -261,7 +261,15 @@ LaserParticleContainer::ContinuousInjection (const RealBox& injection_box)
 #else
     const Real p_pos[1] = {m_updated_position[2]};
 #endif
-    if ( injection_box.contains(p_pos) ){
+#if defined(WARPX_DIM_RZ)
+    // In RZ, check if laser enters the box only along z
```

Comment:
It would make for a long comment, but maybe also say that the special check is needed since "contains" fails in RZ, and explaining how it fails. But otherwise this is good.

Reply:
Thank you, Dave. I changed the comment, let me know if the new comment is clear enough.